fix: gender neutral editorials, closes https://github.com/openid/OpenID4VCI/issues/123 (#143)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

> Credential Issuers often want to know what Wallet they are issuing Credentials to and how private keys are managed for the following reasons:

Author: Credential Issuers needs to know the Wallet that is receiving the Credentials and the security measures in place for managing private keys, for the following reasons:

> Credential Issuers often want to know what Wallet they are issuing Credentials to and how private keys are managed for the following reasons:

Author: ```suggestion
Credential Issuers needs to know the Wallet that is receiving the Credentials and the security measures in place for managing private keys, for the following reasons:
```

Reviewer: the original text honestly sounds ok..

> * The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential Holders by presenting their Credentials.

Author: ```suggestion
* The Credential Issuer MAY want to ensure that private keys are adequately protected against exfiltration and replay. This precaution helps prevent adversaries from impersonating legitimate Credential Holders and presenting the Credentials.
```

Reviewer: the original text is grammatically correct, too :).

Author: I'm neutral, I only have tried to resolve the pending comments. If the authors agreed they may reject or merge this last suggestion, no worries.

## Credential Offer - Same-Device {#use-case-3}

> While browsing the university's home page, the End-User finds a link "request your digital diploma". The End-User clicks on this link and is being redirected to a digital Wallet. The Wallet notifies the End-User that a Credential Issuer offered to issue a diploma Credential. User confirms this inquiry and is taken to the university's Credential issuance service's End-User experience. Upon successful authentication at the university and consent to the issuance of a digital diploma, the End-User is redirected back to the Wallet. Here, the End-User can verify the successful creation of the digital diploma.

Author: ```suggestion
While browsing the university's home page, the End-User finds a link "request your digital diploma". The End-User clicks on this link and is being redirected to a digital Wallet. The Wallet notifies the End-User that a Credential Issuer offered to issue a diploma Credential. End-User confirms this inquiry and is taken to the university's Credential issuance service's End-User experience. Upon successful authentication at the university and consent to the issuance of a digital diploma, the End-User is redirected back to the Wallet. Here, the End-User can verify the successful creation of the digital diploma.
```

## Credential Offer - Cross-Device (with Information Pre-Submitted by the End-User) {#use-case-4}

> The End-User is starting a job at a new employer. The employer requests the End-User to upload specific documents to the employee portal. After a few days, the End-User receives an email from the employer indicating that the employee Credential is prepared and instructing to scan a QR code for its retrieval. The End-User scans the QR code with the smartphone, which opens the Wallet. Meanwhile, the End-User has received a text message with a Transaction Code to the smartphone. After entering that Transaction Code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives Credential into the Wallet.

Reviewer 2: ```suggestion
The End-User is starting a job at a new employer. The employer requests the End-User to upload specific documents to the employee portal. After a few days, the End-User receives an email from the employer indicating that their employee Credential is ready to be claimed and provides instructions to scan a presented QR code for its retrieval. The End-User scans the QR code with their smartphone, which opens the Wallet. Meanwhile, the End-User has received a text message with a Transaction Code to the smartphone. After entering the Transaction Code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives the Credential into their Wallet.
```

> The End-User is starting a job at a new employer. The employer requests the End-User to upload specific documents to the employee portal. After a few days, the End-User receives an email from the employer indicating that their employee Credential is ready to be claimed and provides instructions to scan a presented QR code for its retrieval. The End-User scans the QR code with their smartphone, which opens the Wallet. Meanwhile, the End-User has received a text message with a Transaction Code to the smartphone. After entering the Transaction Code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives the Credential into their Wallet.

Reviewer 3: See #123 for using "their" to replace a singular pronoun.

Author: Done, thx

> The End-User is starting a job at a new employer. The employer requests the End-User to upload specific documents to the employee portal. After a few days, the End-User receives an email from the employer indicating that their employee Credential is ready to be claimed and provides instructions to scan a presented QR code for its retrieval. The End-User scans the QR code with their smartphone, which opens the Wallet. Meanwhile, the End-User has received a text message with a Transaction Code to the smartphone. After entering the Transaction Code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives the Credential into their Wallet.

Author: ```suggestion
The End-User is starting a job at a new employer. The employer requests the End-User to upload specific documents to the employee portal. After a few days, the End-User receives an email from the employer indicating that the employee Credential is ready to be claimed and provides instructions to scan a presented QR code for its retrieval. The End-User scans the QR code with the smartphone, which opens the Wallet. Meanwhile, the End-User has received a text message with a Transaction Code to the smartphone. After entering the Transaction Code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives the Credential into the Wallet.
```

## Credential Offer - Cross-Device & Deferred {#use-case-5}

> The End-User intends to acquire a digital criminal record. This involves a visit to the local administration's office to request the official criminal record be issued as a digital Credential. After presenting the ID document, the End-User is prompted to scan a QR code using the Wallet. It is communicated that the actual issuance of the Credential will require some time due to necessary background checks by the authority.

Author: @ju-cu I can buy this while at the same time I'm wondering that the term officials may intend humans while there might be automated machines for id proofing and issuance

Nothing prevent to call officials also machines ... Just a though, I'm keeping my hand to acquire this suggestion. Thx!

> The End-User intends to acquire a digital criminal record. This involves a visit to the local administration's office to request the official criminal record be issued as a digital Credential. After presenting the ID document, the End-User is prompted to scan a QR code using the Wallet. It is communicated that the actual issuance of the Credential will require some time due to necessary background checks by the authority.

Reviewer 3: Hm... Still wondering who communicates :D I don't have a strong opinion on that small editorial change.  Personally, I prefer "inform the End-User" over "is communicated" as the latter does neither include the sender nor receiver of the information (at least not in the current version). Maybe "the End-User is informed..."?

> The End-User intends to acquire a digital criminal record. This involves a visit to the local administration's office to request the official criminal record be issued as a digital Credential. After presenting the ID document, the End-User is prompted to scan a QR code using the Wallet. It is communicated that the actual issuance of the Credential will require some time due to necessary background checks by the authority.

Author: ```suggestion
The End-User intends to acquire a digital criminal record. This involves a visit to the local administration's office to request the official criminal record be issued as a digital Credential. After presenting the ID document, the End-User is prompted to scan a QR code using the Wallet and is informed that the issuance of the Credential will require some time, due to necessary background checks by the authority.
```

> The End-User, during the usage of the Wallet, notices a sign that the issuance of the digital record is in progress. After a few days, a notification is received from the Wallet indicating that the requested Credential has been successfully issued. Upon opening the Wallet, the End-User is queried about the download of the Credential. After confirmation, the new Credential is fetched and saved in the Wallet.

Reviewer 3: I actually prefer some parts of the old description. 
Grammar: "has been" => "was"

```suggestion
While using the Wallet, the End-User notices an indication that the issuance of the digital record is in progress. After a few days, the End-User receives a notification from the Wallet indicating that the requested Credential was successfully issued. Upon opening the Wallet, the End-User is queried about the download of the Credential. After confirmation, the Wallet fetches and saves the new Credential.
```

## Wallet Initiated Issuance during Presentation {#use-case-1}

> An End-User comes across a verifier app that is requesting the End-User to present a Credential, e.g., a driving license. The Wallet determines the requested Credential type(s) from the presentation request and notifies the End-User that there is currently no matching Credential in the Wallet. The Wallet selects a Credential Issuer capable of issuing the lacking Credential and, upon End-User consent, sends the End-User to the Credential Issuer's End-User experience (Web site or app). Once authenticated and consent is provided for the issuance of the Credential into the Wallet, the End-User is redirected back to the Wallet. The Wallet informs the End-User that Credential was successfully issued into the Wallet and is ready to be presented to the verifier app that originally requested presentation of that Credential.

Reviewer 2: ```suggestion
An End-User comes across a verifier app that is requesting the End-User to present a Credential, e.g., a driving license. The Wallet determines the requested Credential type(s) from the presentation request and notifies the End-User that there is currently no matching Credential in the Wallet. The Wallet selects a Credential Issuer capable of issuing the missing Credential and, upon End-User consent, sends the End-User to the Credential Issuer's End-User experience (Web site or app). Once authenticated and consent is provided for the issuance of the Credential into the Wallet, the End-User is redirected back to the Wallet. The Wallet informs the End-User that Credential was successfully issued into the Wallet and is ready to be presented to the verifier app that originally requested presentation of that Credential.
```
